src/main.rs

```diff
@@ -166,6 +166,12 @@ fn main() {
                             .long("default")
                             .help("Print the default instead of the computed config")
                             .takes_value(false),
+                    )
+                    .arg(
+                        Arg::with_name("path")
+                            .help("Configuration section or value to print")
```

Comment:
I think this should be plural, and I don't think value quite fits. Maybe subsets would be a better term?

Reply:
I changed the argument name to "name" and its help to "keys" to match the `config` command.